### PR TITLE
Added SD card layout for both Tiramisu and Aroma

### PR DESCRIPTION
**Pull request by Luca06Luwa**
Not sure if this is a good visualizer for the SD card file structure, but I decided to make a PR anyways.

### CODE REVIEW

**Comment by Lazr1026** [comment]
I like the layout and how the tree is shown but i have a few issues with it:

Tiramisu:
> All other apps like disc2app, nanddumper, etc. should be here too)

disc2app and nanddumper do not work in Tiramisu. i think it would be best to not mention any apps on this page as i feel it will confuse the user.

Aroma:
> All of the aroma modules ending with.wms like AromaBaseModule, WUHBUtilityModule, etc. should be here too)

Missing a space between "with" and "`.wms`". Same thing about the apps i mention in the Tiramisu file tree applies here too. Also capitalize "Aroma"

**Comment by Luca06Luwa** [comment]
Should be fixed now.

**Comment by Lazr1026** [comment]
Okay i think it looks good now, will wait for anyone with review/merge access.

File: docs/user-guide/aroma/sd-preparation.md
Comment (GaryOderNichts) [inline docs/user-guide/aroma/sd-preparation.md:36]: I don't think we need to list the `homebrew_launcher` in the aroma structure, this might only lead to confusion.

File: docs/user-guide/aroma/sd-preparation.md
Comment (GaryOderNichts) [inline docs/user-guide/aroma/sd-preparation.md:56]: Why is there a `99_autoboot.rpx` in the `plugins` folder?

File: docs/user-guide/aroma/sd-preparation.md
Comment (GaryOderNichts) [inline docs/user-guide/aroma/sd-preparation.md:70]: I don't think it is necessary to list the `installer` and `tiramisu` environment for the aroma structure.

File: docs/user-guide/aroma/sd-preparation.md
Comment (GaryOderNichts) [inline docs/user-guide/aroma/sd-preparation.md:75]: I don't think the `fw_img_loader` is part of the download by default?

File: docs/user-guide/aroma/sd-preparation.md
Comment (GaryOderNichts) [inline docs/user-guide/aroma/sd-preparation.md:48]: The aroma guide currently doesn't add sigpatches to the environment.

**Comment by Luca06Luwa** [comment]
I have gone ahead and merged the changes that were suggested.

File: docs/user-guide/aroma/sd-preparation.md
Comment (Lazr1026) [inline docs/user-guide/aroma/sd-preparation.md:39]: Missing an "s".

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

File: docs/user-guide/aroma/sd-preparation.md
Comment (Lazr1026) [reply docs/user-guide/aroma/sd-preparation.md:39]: This is also an issue on the Tiramisu sd-preparation.

**Review by Lazr1026 — APPROVED** [review]

File: docs/user-guide/aroma/sd-preparation.md
Comment (Lazr1026) [inline docs/user-guide/aroma/sd-preparation.md:51]: `regionfree.rpx` is not a thing. i think you meant `regionfree.wps`

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

File: docs/user-guide/aroma/sd-preparation.md
Comment (Luca06Luwa) [reply docs/user-guide/aroma/sd-preparation.md]: Yeah I had forgot to rename it

File: docs/user-guide/aroma/sd-preparation.md
Comment (Lazr1026) [inline docs/user-guide/aroma/sd-preparation.md:51]: Add a note about plugins here. 

Example: `(All other Aroma plugins ending with .wps should be here too)`

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

**Comment by Luca06Luwa** [comment]
I noticed the sd card layout broke one of the info bubbles. #168 hopefully should fix it.

**Pull request merged**
